docs/k0s-install.md

```diff
@@ -0,0 +1,306 @@
+INFO [ssh] 145.40.96.1:22: is running Ubuntu 20.10 
+INFO ==> Running phase: Prepare hosts    
+INFO [ssh] 145.40.96.3:22: installing kubectl     
+INFO ==> Running phase: Gather host facts 
+INFO [ssh] 145.40.96.3:22: discovered bond0 as private interface 
+INFO [ssh] 145.40.96.3:22: discovered 10.12.18.133 as private address 
+INFO ==> Running phase: Validate hosts   
+INFO ==> Running phase: Gather k0s facts 
+INFO ==> Running phase: Download K0s on the hosts 
+INFO [ssh] 145.40.96.1:22: downloading k0s 0.10.0 
+INFO [ssh] 145.40.96.3:22: downloading k0s 0.10.0 
+INFO ==> Running phase: Configure K0s    
+WARN [ssh] 145.40.96.3:22: generating default configuration 
+INFO [ssh] 145.40.96.3:22: validating configuration 
+INFO [ssh] 145.40.96.3:22: configuration was changed 
+INFO ==> Running phase: Initialize K0s Cluster 
+INFO [ssh] 145.40.96.3:22: installing k0s controller 
+INFO [ssh] 145.40.96.3:22: waiting for the k0s service to start 
+INFO [ssh] 145.40.96.3:22: waiting for kubernetes api to respond 
+INFO ==> Running phase: Install workers  
+INFO [ssh] 145.40.96.3:22: generating token       
+INFO [ssh] 145.40.96.1:22: writing join token     
+INFO [ssh] 145.40.96.1:22: installing k0s worker  
+INFO [ssh] 145.40.96.1:22: starting service       
+INFO [ssh] 145.40.96.1:22: waiting for node to become ready 
+INFO ==> Running phase: Disconnect from hosts 
+INFO ==> Finished in 2m2s                
+INFO k0s cluster version 0.10.0 is now installed  
+INFO Tip: To access the cluster you can now fetch the admin kubeconfig using: 
+INFO      k0sctl kubeconfig              
+```
+
+And, -- presto you k0s cluster is up and running
+
+Get kubeconfig:
+```
+$ k0sctl kubeconfig > kubeconfig
+$ KUBECONFIG=kubeconfig kubectl get pods -A
+NAMESPACE     NAME                                       READY   STATUS    RESTARTS   AGE
+kube-system   calico-kube-controllers-5f6546844f-w8x27   1/1     Running   0          3m50s
+kube-system   calico-node-vd7lx                          1/1     Running   0          3m44s
+kube-system   coredns-5c98d7d4d8-tmrwv                   1/1     Running   0          4m10s
+kube-system   konnectivity-agent-d9xv2                   1/1     Running   0          3m31s
+kube-system   kube-proxy-xp9r9                           1/1     Running   0          4m4s
+kube-system   metrics-server-6fbcd86f7b-5frtn            1/1     Running   0          3m51s
+```
+
+### Upgrade k0s version on your setup
+
+_NOTE: Upgrade will only work if you previosuly used k0sctl v0.4.0 and you are using v0.5.0_
+
+The previous example installed k0s v0.10.0 on one controller and one worker.
+Let's upgrade the cluster to `v0.11.0-beta.2` 
+Note: to get most recent version please visit release [page](https://github.com/k0sproject/k0s/tags).
+
+In the k0sctl.yaml change this yaml tag:
+```
+  k0s:
+    version: 0.11.0-beta.2
+```
+Make sure to omit `v` in version name
+
+```
+$ k0sctl apply --config /root/go/src/github.com/k0sproject/k0s/k0sctl.yaml 
```

Comment:
Probably better to use some nicer path (or drop the `--config` and assume `./k0sctl.yaml`.